src/main/webapp/script.js

```diff
@@ -110,6 +110,719 @@ function createCommentElement(comment) {
   return commentElement;
 }
 
+/** Gets the ID of the YouTube video that the user inputs. */
+function getId() {
+  const liveStreamLink = document.getElementById('live-stream-link').value;
+  return getIdFromUrl(liveStreamLink);
+}
+
+/** Gets the ID of the YouTube video that the user inputs. */
+function storeLiveStreamInfo() {
+  const liveStreamLink = document.getElementById('live-stream-link').value;
+  return false;
+}
+
+/** Gets the ID of a YouTube video from its URL.
+    Will work only for a URL of a certain format
+    (i.e. youtube.com/watch?v=). */
+function getIdFromUrl(url) {
```

Comment:
Is this the correct way of doing this? What if the url has other parameters such as hl?
I think the correct method would be split anything after '?' on '&' and then split those for key-value pairs or alternatively identify the version key and then its value.

Reply:
+1
I think URLSearchParams would be useful here again

Reply:
Based on testing, it appears that URLSearchParams relies on the ampersand and some YouTube URLs dont have it :(

Reply:
Changing to use URLSearchParams if there's an ampersand (i.e. there are multiple param vals); if there are no &, it is assumed that v= is the only such parameter value. Further suggestions are welcome!